elasticsearch-core/src/test/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClientTest.scala

```diff
@@ -560,6 +560,21 @@ class RestlasticSearchClientTest extends WordSpec with Matchers with ScalaFuture
       matchQueryFuture.futureValue.sourceAsMap.toSet should be(Set(Map("f1" -> "multimatch1 test", "f2" -> 1, "text" -> "text1")))
     }
 
+    "support disjuntion max query" in {
+      // https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-dis-max-query.html
+      val disMaxDoc1 = Document("disMaxDoc1", Map("title" -> "Quick brown rabbits", "body" -> "Brown rabbits are commonly seen"))
+      val disMaxDoc2 = Document("disMaxDoc2", Map("title" -> "Keeping pets healthy", "body" -> "My quick brown fox eats rabbits on a regular basis"))
+      val docsFuture = restClient.bulkIndex(index, tpe, Seq(disMaxDoc1, disMaxDoc2))
+      whenReady(docsFuture) { _ => refresh() }
+
+      val disMaxQuery = DisMaxQuery(queries = Seq(MatchQuery("title", "Brown fox"),MatchQuery("body", "Brown fox")))
```

Comment:
can you set the tie breaker and boost fields in the test to get slightly better coverage?